Buildings/ThermalZones/Detailed/Validation/BESTEST/Data/CriteriaLimits.mo

```diff
@@ -0,0 +1,21 @@
+within Buildings.ThermalZones.Detailed.Validation.BESTEST.Data;
+record CriteriaLimits
+  "Record that is used for summary of the test acceptance criteria limits"
+  extends Modelica.Icons.Record;
+  parameter Real LowerLimit(unit="J") "Lower limit of the test acceptance criteria";
+  parameter Real UpperLimit(unit="J") "Upper limit of the test acceptance criteria";
```

Comment:
I think we can just use Modelica.Units.SI.Energy rather than declaring the `unit` attribute (which if declared should be `final`).
Also, please use lower case `lowerLimit` to follow the conventions.